pkg/cli/common/util.go

```diff
@@ -102,3 +109,21 @@ func WriteConfigFile() error {
 	}
 	return nil
 }
+
+// StringInSlice verifies if a string slice contains a string value
+func StringInSlice(s string, slice []string) bool {
```

Comment:
This is already used in `pkg/core/extension_store.go`. Could you move the method to common place which could be imported from anywhere in core code?

Reply:
BTW there's also `contains` method (same thing) declared at 3 different places in core...

Reply:
took a lot of changes, but I moved it to a common util package

Reply:
great!